performance/software.md

```diff
@@ -24,6 +24,8 @@ It's critical to understand your particular model size and framework requirement
 
 If your network is very slow, your training is likely to be network-bound and many improvements in the training setup will not help with the improving performance.
 
+Note: The [cookbook](https://github.com/EleutherAI/cookbook) contains a set of [communication benchmarks](https://github.com/EleutherAI/cookbook/tree/main/benchmarks/communication) for each collective that you can use to quickly measure the throughput of your internode or intranode network.
```

Comment:
can we move it to https://github.com/stas00/ml-engineering/tree/master/network 

I need to do a lot more massaging to make everything well organized.

Reply:
will move

Reply:
Hmm, I will need to rework things first, it's perfect where you placed it in the first place until then. So nothing to change.